reserve_harvester.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+A script to gather Wikidata IDs of Swedish nature reserves.
+
+The input is a Petscan dump from svwp category "Naturreservat i Sverige",
+two levels deep (naturreservat -> nr i <län> -> nr i <kommun>):
+https://petscan.wmflabs.org/?psid=914993
+
+Then it's compared to NR_polygon.csv the following way:
+
+* The municipalities of each wp article are extracted
+* If a reserve in the csv file has exactly the same set of municipalities,
+as well as an identical or very similar name (Foo / Foo natturreservat),
+it's added to the exact matches file: svwp_to_nature_id_exact.json
+* If more than one match is found, they're added to
+svwp_to_nature_id_multiple.json, which can be reviewed manually.
+* If there's no match, the svwp article is added to
+svwp_to_nature_id_none, which can be reviewed manually.
+
+When running, if an svwp article with no WD item is encountered,
+it is created automatically. Thus, all the articles in the mapping
+files have corresponding WD items.
+
+CAVEATS
+=======
+
+Sometimes a reserve described in one article has two
+separate nature IDs, when it spans across several
+counties, eg. Sandsjöbacka naturreservat is two separate reserves
+in VG and Halland. How to handle those?
```

Comment:
I don't think there's a lot of cases like this. I've been thinking about this solution and I guess it depends on whether the area counts as one reserve or several, legally. If you look at http://www.lansstyrelsen.se/VastraGotaland/Sv/djur-och-natur/skyddad-natur/naturreservat/lanets-naturreservat/molndal/sandsjobacka/Pages/index.aspx, it has one operator, even though both Göteborg (VG) and Kungsbacka (H) are listed as municipalities.